api/server/runner.go

```diff
@@ -20,24 +20,35 @@ type runnerResponse struct {
 	Error     *models.ErrorBody `json:"error,omitempty"`
 }
 
-func (s *Server) handleRequest(c *gin.Context) {
-	if strings.HasPrefix(c.Request.URL.Path, "/v1") {
-		c.Status(http.StatusNotFound)
-		return
-	}
-
-	r, routeExists := c.Get(api.Path)
-	if !routeExists {
-		r = "/"
+// handleFunctionCall executes the function.
+// Requires the following in the context:
+// * "app_name"
+// * "path"
+func (s *Server) handleFunctionCall(c *gin.Context) {
+	// @treeder: Is this necessary? An app could have this prefix too. Leaving here for review.
```

Comment:
it seems like a pretty silly guard, so +1 remove. the front end had some funny business where this could have potentially been called from somewhere where this were true, because it was falling back any request where the gin router didn't match to try to run a function call with that path, but this has been removed now.